ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -558,6 +558,13 @@ private static void createPhaseTapChanger(UcteAngleRegulation ucteAngleRegulatio
                 .setRegulationValue(ucteAngleRegulation.getP())
                 .setRegulationMode(PhaseTapChanger.RegulationMode.FIXED_TAP);
 
+        if (!Double.isNaN(ucteAngleRegulation.getP())) {
+            ptca.setRegulationMode(PhaseTapChanger.RegulationMode.ACTIVE_POWER_CONTROL)
+                    .setTargetDeadband(0.0)
+                    .setRegulationTerminal(transformer.getTerminal1())
+                    .setRegulating(true);
```

Comment:
@TheMaskedTurtle maybe in order to not loose this PR, I can set the regulating boolean to false here? Or I close the PR, as you prefer.

Reply:
@annetill ok you can set regulation to false, to at least import the correct values from the UCTE file.